pages/manual/contributing.md

```diff
@@ -32,3 +32,13 @@ It is recommended to use [ClangFormat](http://clang.llvm.org/docs/ClangFormat.ht
 
 ### Contributing whole modules
 If you are working on a large Inviwo Module and possibly have dependencies on big external libraries, we have an extra [modules repository](https://github.com/inviwo/modules).
+
+#### Inviwo Marketplace
```

Comment:
Nit: is marketplace the right name? Will users be able to sell modules? otherwise I think something like "Inviwo Module Index" would be more suitable

Reply:
Agree about the marketplace, even though things can be for free. I still like "Marketplace" as it is quite general and to the point.
Other alternatives that I can think of:
Inviwo Module Repository - conflicts a bit with the "extra module repository", but we might call that something else.
Inviwo (Module) Packages - indicates that it enables retrieval of pre-built modules

Reply:
I guess it can be considered a form of package manager for Inviwo...

Reply:
python pip's pypi uses Python Package Index. Think that makes much more sense than calling it a marketplace. If I here marketplace I expect that I will have to pay to done load things. Repository is too overloaded.

Reply:
Microsoft VS Code uses the term Marketplace for their extensions, but they also mark them as "free". I don't know if you can even pay for extensions there...

Inviwo Package Index sounds ok to me. A bit verbose with the "Index" in my opinion.

Reply:
You need to pay for quite many of the VS extensions. 
Inviwo Package Index, is ok I guess, although I find "package" quite vague, and overloaded. At least in the context of Inviwo development "module" is a somewhat well defined concept. 
I guess the same vagueness applies to "Index". Think Index make sense if we are not actually hosting the modules, but just providing links/easy access to them. Similar to what pipy does. Not sure how we actually plan to do that?